src/kOS/Binding/FlightControlManager.cs

```diff
@@ -45,176 +41,80 @@ public override void AddTo(SharedObjects shared)
             SafeHouse.Logger.Log("FlightControlManager.AddTo " + Shared.Vessel.id);
 
             currentVessel = shared.Vessel;
-            ConnectivityManager.AddAutopilotHook(currentVessel, OnFlyByWire);
 
-            AddNewFlightParam("throttle", Shared);
-            AddNewFlightParam("steering", Shared);
-            AddNewFlightParam("wheelthrottle", Shared);
-            AddNewFlightParam("wheelsteering", Shared);
+            shared.BindingMgr.AddBoundVariable("THROTTLE", GetThrottleValue, SetThrottleValue);
+            shared.BindingMgr.AddBoundVariable("STEERING", GetSteeringValue, SetSteeringValue);
+            shared.BindingMgr.AddBoundVariable("WHEELSTEERING", GetWheelSteeringValue, SetWheelSteeringValue);
+            shared.BindingMgr.AddBoundVariable("WHEELTHROTTLE", GetSteeringValue, SetSteeringValue);
```

Comment:
I got an error when I tried using ``lock wheelthrottle``, where it claimed I can't use a Scalar for it (because it thought I was setting the steering instead).  This looks like a cut and paste error since wheelthrottle appears to be setting the same thing as steering does.